com.unity.netcode.adapter.utp/Runtime/com.unity.netcode.adapter.utp.asmdef

```diff
@@ -15,6 +15,17 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.multiplayer.tools",
+            "expression": "",
+            "define": "MULTIPLAYER_TOOLS"
+        },
+        {
+            "name": "com.unity.multiplayer.tools",
+            "expression": "1.0.0-pre.3",
```

Comment:
Is this intentionally 1.0.0-pre.3 or should it be 1.0.0-pre.4?  (just checking)

Reply:
That is a mistake :) We bumped to pre-4 in the mean time and I forgot to update this